alacritty/src/renderer/mod.rs

```diff
@@ -209,8 +216,9 @@ impl GlyphCache {
 
     fn load_glyphs_for_font<L: LoadGlyph>(&mut self, font: FontKey, loader: &mut L) {
         let size = self.font_size;
-        for i in 32u8..=126u8 {
-            self.get(GlyphKey { font_key: font, c: i as char, size }, loader);
+        for character in 32u8..=126u8 {
+            let character = character as char;
+            self.get(GlyphKey { font_key: font, character, size }, loader);
```

Comment:
I don't think this is an improvement at all? `32u8..=126u8` doesn't return a character, so `i` is more appropriate.